neo/IO/Data/RocksDB/RocksDBCore.cs

```diff
@@ -0,0 +1,181 @@
+using RocksDbSharp;
+using System;
+using System.Collections.Generic;
+
+namespace Neo.IO.Data.RocksDB
+{
+    public class RocksDBCore : IDisposable
+    {
+        public static readonly Options OptionsDefault = new Options();
+        public static readonly ReadOptions ReadDefault = new ReadOptions();
+        public static readonly WriteOptions WriteDefault = new WriteOptions();
+        public static readonly WriteOptions WriteDefaultSync = new WriteOptions();
+
+        #region Families
+
+        private const string DATA_Block_Name = "Block";
+        private const string DATA_Transaction_Name = "Transaction";
+
+        private const string ST_Contract_Name = "Contract";
+        private const string ST_Storage_Name = "Storage";
+
+        private const string IX_HeaderHashList_Name = "HeaderHashList";
+        private const string IX_CurrentBlock_Name = "CurrentBlock";
+        private const string IX_CurrentHeader_Name = "CurrentHeader";
+
+        private const string SYS_Version_Name = "Version";
+
+        internal readonly ColumnFamilyHandle DATA_Block;
+        internal readonly ColumnFamilyHandle DATA_Transaction;
+
+        internal readonly ColumnFamilyHandle ST_Contract;
+        internal readonly ColumnFamilyHandle ST_Storage;
+
+        internal readonly ColumnFamilyHandle IX_HeaderHashList;
+        internal readonly ColumnFamilyHandle IX_CurrentBlock;
+        internal readonly ColumnFamilyHandle IX_CurrentHeader;
+
+        internal readonly ColumnFamilyHandle SYS_Version;
+
+        internal readonly ColumnFamilyHandle DefaultFamily;
+
+        #endregion
+
+        static RocksDBCore()
+        {
+            WriteDefaultSync.SetSync(true);
+        }
+
+        private readonly RocksDb _rocksDb;
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="db">Database</param>
+        private RocksDBCore(RocksDb db)
```

Comment:
This is a wrapper?

Reply:
Yes, has the same logic as LevelDBCore

Reply:
I think it's not the same. `LevelDBCore` is a wrapper of the handle while `RocksDBCore` is a wrapper of `RocksDb`, which is another wrapper of the handle.

Reply:
RocksDbCore exists because is required for DbCache and DbMetaCache, so if your proposal is to use only RocksDbStore, i will have a circular dependency with DbCache and DbMetaCache